website/src/components/TeamProfileCards/index.tsx

```diff
@@ -40,9 +41,9 @@ function TeamProfileCard({
       <div className="card card--full-height">
         <div className="card__header">
           <div className="avatar avatar--vertical">
-            <img
+            <Image
               className="avatar__photo avatar__photo--xl"
-              src={`${githubUrl}.png`}
+              img={`${githubUrl}.png`}
```

Comment:
Ideal image is not designed to work with external URLs. `<img>` is fine in this case